.circleci/workflows.yml

```
@@ -1221,17 +1228,24 @@ workflows:
             branches:
               only:
                 - main
+      - gcp-gcr/build-and-push-image:
+          context: dataeng-bqetl-gcr
+          dockerfile: DSRETest.Dockerfile
+          image: bigquery-etl
+          tag: DSRE-1683-test
+
```

Guessing if you add this the [build-and-push-image](https://circleci.com/developer/orbs/orb/circleci/gcp-gcr#jobs-build-and-push-image) step will fail the same way the `deploy-to-private-gcr` step is failing

```suggestion
          use_oidc: true
```